src/App/Options/Cmd/KafkaToSqs.hs

```diff
@@ -155,36 +148,6 @@ onRebalance lgr stats e = case e of
     S.sendEvt stats $ S.event "Rebalancing" partitionsText
   _ -> pure ()
 
-sendSqsC :: (MonadAWS m, MonadResource m)
-  => T.Text
-  -> ConduitT J.Value () m ()
-sendSqsC queueUrl = mapMC
-  $ sendSqs queueUrl
-  . T.pack
-  . C8.unpack
-  . toStrict
-  . J.encode
-
-transmitOneC :: Monad m => ConduitT a a m ()
-transmitOneC = C.await >>= mapM_ yield
-
-backPressure :: (MonadAWS m, MonadResource m, MonadLogger m) => T.Text -> Int -> ConduitT a a m ()
-backPressure queueUrl maxMessages = go 0
-  where go n = if n > 0
-          then do
-            transmitOneC
-            go 0
-          else do
-            maybeNumMessages <- lift $ getSqsApproximateNumberOfMessages queueUrl
-            case maybeNumMessages of
-              Just numMessages -> if numMessages > maxMessages
-                then do
-                  liftIO $ threadDelay 1000000 -- µs
-                  logInfo $ "Target queue " <> show queueUrl <> " is full (" <> show numMessages <> " messages)"
-                  go 0
-                else go (maxMessages - numMessages)
```

Comment:
But potentially, a hard loop can happen here.  It might be the case we have a bug in the `backPressure` function.

Reply:
Do you think we should fix `backPressure`, or not use it?